diff --git a/src/Helpers/routes_helpers.php b/src/Helpers/routes_helpers.php
index 77ba9199..06cbe23f 100644
--- a/src/Helpers/routes_helpers.php
+++ b/src/Helpers/routes_helpers.php
@@ -1,7 +1,7 @@
 <?php
 
-use Illuminate\Support\Facades\Request;
 use Illuminate\Support\Facades\Route;
+use Illuminate\Support\Arr;
 use Illuminate\Support\Str;
 
 if (! function_exists('moduleRoute')) {
@@ -15,6 +15,9 @@ if (! function_exists('moduleRoute')) {
         // Fix module name case
         $moduleName = Str::camel($moduleName);
 
+        // Wrap scalar parameters in an array
+        $parameters = Arr::wrap($parameters);
+
         // Nested module, pass in current parameters for deeply nested modules
         if (Str::contains($moduleName, '.')) {
             if (Str::endsWith(request()->route()->getName(), 'restoreRevision')) {
