diff --git a/src/Repositories/ModuleRepository.php b/src/Repositories/ModuleRepository.php
index 47e56466..5172e25d 100644
--- a/src/Repositories/ModuleRepository.php
+++ b/src/Repositories/ModuleRepository.php
@@ -696,6 +696,8 @@ abstract class ModuleRepository
         if (! $modelOrRepository) {
             if (class_exists($relation) && (new $relation()) instanceof Model) {
                 $modelOrRepository = Str::afterLast($relation, '\\');
+            } elseif (class_exists($relation) && (new $relation()) instanceof \App\Libraries\Api\Models\BaseApiModel) {
+                $modelOrRepository = 'Api\\' . Str::afterLast($relation, '\\');
             } else {
                 $morphedModel = Relation::getMorphedModel($relation);
                 if (class_exists($morphedModel) && (new $morphedModel()) instanceof Model) {
