diff --git a/src/Repositories/ModuleRepository.php b/src/Repositories/ModuleRepository.php
index cb6986e0..bf268cc9 100644
--- a/src/Repositories/ModuleRepository.php
+++ b/src/Repositories/ModuleRepository.php
@@ -867,6 +867,8 @@ abstract class ModuleRepository
         if (!$modelOrRepository) {
             if (class_exists($relation) && (new $relation) instanceof Model) {
                 $modelOrRepository = str_after_last($relation, '\\');
+            } elseif (class_exists($relation) && (new $relation) instanceof \App\Libraries\Api\Models\BaseApiModel) {
+                $modelOrRepository = 'Api\\' .str_after_last($relation, '\\');
             } else {
                 $morphedModel = Relation::getMorphedModel($relation);
                 if (class_exists($morphedModel) && (new $morphedModel) instanceof Model) {
