diff --git a/frontend/js/libs/Quill/QuillConfiguration.js b/frontend/js/libs/Quill/QuillConfiguration.js
index df8934b9..bc145ebb 100644
--- a/frontend/js/libs/Quill/QuillConfiguration.js
+++ b/frontend/js/libs/Quill/QuillConfiguration.js
@@ -119,8 +119,7 @@ class MyLink extends Link {
 
     // url starting with the front-end base url wont have target blank
     if (window[process.env.VUE_APP_NAME].STORE.form.baseUrl) {
-      const url = new URL(window[process.env.VUE_APP_NAME].STORE.form.baseUrl)
-      if (value.startsWith(url.origin)) {
+      if (value.startsWith(window[process.env.VUE_APP_NAME].STORE.form.baseUrl)) {
         node.removeAttribute('target')
       }
     }
