diff --git a/src/Services/Listings/Columns/Relation.php b/src/Services/Listings/Columns/Relation.php
index 9c6c05ec..6aa2fdd2 100644
--- a/src/Services/Listings/Columns/Relation.php
+++ b/src/Services/Listings/Columns/Relation.php
@@ -5,6 +5,7 @@ namespace A17\Twill\Services\Listings\Columns;
 use A17\Twill\Exceptions\ColumnMissingPropertyException;
 use A17\Twill\Models\Contracts\TwillModelContract;
 use A17\Twill\Services\Listings\TableColumn;
+use Illuminate\Support\Collection;
 use Illuminate\Support\Str;

 class Relation extends TableColumn
@@ -44,9 +45,9 @@ class Relation extends TableColumn
             throw new ColumnMissingPropertyException('Relation column missing relation value: ' . $this->field);
         }

-        /** @var \Illuminate\Database\Eloquent\Collection $relation */
         $model->loadMissing($this->relation);
-        $relation = collect($model->getRelation($this->relation));
+        /** @var \Illuminate\Database\Eloquent\Collection $relation */
+        $relation = Collection::wrap($model->getRelation($this->relation));

         return $relation->pluck($this->field)->join(', ');
     }
